sage.py

```diff
@@ -1629,126 +1425,111 @@ def make_AG(condensed_v_data, condensed_data, state_groups, sev_sinks, datafile,
     #    print(attackerID, len(v), set([x.split('|')[0] for x in v]))
 
 
-
-
-## ----- main ------    
+# ----- MAIN ------
 
 if len(sys.argv) < 5:
-    print('USAGE: sage.py {path/to/json/files} {experiment_name} {alert_filtering_window (def=1.0)} {alert_aggr_window (def=150)} {(start_hour,end_hour)[Optional]}')
```

Comment:
Usually, Unix man-pages have "Usage", that's why I have changed it

Reply:
A better idea: replacing this manually written code with ConfigParser (see example in SECLEDS). Makes it easier to handle the command line params.

Reply:
Do you mean `ArgumentParser`? I think `ConfigParser` is for an init file (in either case, this can be left for another PR)

![image](https://github.com/tudelft-cda-lab/SAGE/assets/45541174/1002e6ea-67fa-4dcc-b136-b92a17512b61)

Reply:
I have created an issue for this dataset part, you might check it and comment if necessary. I will resolve all other comments related to this issue